aws/resource_aws_fsx_lustre_file_system_test.go

```diff
@@ -977,14 +988,14 @@ resource "aws_fsx_lustre_file_system" "test" {
 `, retention)
 }
 
-func testAccAwsFsxLustreFileSystemDeploymentType(deploymentType string) string {
```

Comment:
Nit: Over time we have found that configurable test configuration functions like these are more useful and require less maintenance. Would recommend reverting this back and just calling the function with the parameter set to `SCRATCH_1` or otherwise renaming the test function as `testAccAwsFsxLustreFileSystemDeploymentTypeScratch1()` to signal its value specificity.